internal/api/e2e/framework/server.go

```diff
@@ -36,6 +36,25 @@ import (
 
 var useSQL = os.Getenv("PERSES_TEST_USE_SQL")
 
+func DefaultConfig() config.Config {
+	projectPath := test.GetRepositoryPath()
+	var activatePermission = false
+	return config.Config{
+		Security: config.Security{
+			Readonly:      false,
+			Authorization: config.AuthorizationConfig{EnableAuthorization: &activatePermission},
+			EncryptionKey: promConfig.Secret(hex.EncodeToString([]byte("=tW$56zytgB&3jN2E%7-+qrGZE?v6LCc"))),
+		},
+		Schemas: config.Schemas{
```

Comment:
perhaps we could mutualise the schema config with the one in the `rbac_test` file